docs/contributing/make_changes_through_github.rst

```diff
@@ -0,0 +1,68 @@
+.. _contributing_make_changes_through_github:
+
+
+Making changes directly through Github
+==========================================
+
+If you want to change documentation, but not code, we suggest using the GitHub markdown editor, which means you don’t have to fork the repo at all. Here’s how you do this:
+
+Start editing
+---------------------
+
+**1A. Edit from docs.greatexpecations.io**
+
+    * Go to the `Great Expectations docs <http://docs.greatexpectations.io/en/latest/>`__.
+    * On each page, you’ll see an ``Edit on GitHub`` button in the top right. Click this to go to the source file in the Great Expectations GitHub repo.
+    
+**1B. Edit from Github**
+
+    * If you’re already on GitHub, the docs are located in ``great_expectations > docs``. You can directly navigate to the respective page you want to edit (but getting there from `docs.great_expectations.io <http://docs.greatexpectations.io/en/latest/>`__ is a little easier).
+    * In the top right of the grey header bar of the actual file, click the pencil icon to get into edit mode on GitHub.
+
+**2. Make edits**
+
+    * Make your edits and use the Preview tab to preview changes.
+    * Please pay close attention to the :ref:`contributing_style_guide`.
+
+Submit a pull request
+-------------------------
+
+
+**3. Submit your edits as a PR**
+
+    * When you’re done, add a meaningful commit message at the bottom. Use a short title and a meaningful explanation of what you changed and why.
+    * Click the ``Propose File Change`` button at the bottom of the page.
+    * Click the ``Create Pull Request`` button.
+    * Optionally: Add comment to explain your change, if it’s not already in the commit message.
+    * Click the next ``Create Pull Request`` button to create the actual PR.
+
+
+**4. Sign the CLA**
+
+    * If this is your first contribution to Great Expectations, You will see a comment from the "CLA Bot" that asks you to complete the Contributor Licence Agreement form.
+    * Please complete the form and comment on the PR to say that you’ve signed the form.
+
+**5. Verify continuous integration checks**
+
+    * Wait for the other Continuous Integration (CI) checks to go green and watch out for a comment from the automated linter that checks for syntax and formatting issues.
+    * Fix any issues that are flagged. (For documentation changes, it's unlikely that you'll have any issues.)
+
+**6. Wait for a core team member to approve and merge your PR**
+
+    * Once all checks pass, a Great Expectations team member will approve your PR and merge it. 
+    * GitHub will notify you of comments or a successful merge according to your notification settings.
+    * If there are any issues, please address them promptly.
+
+**7. Do a victory dance**
+
+    * Congratulations! You’ve just contributed to Great Expectations!
+
+        .. image:: great_expectations_running.gif
+
+Video instructions
+------------------
+
+#FIXME: Insert video version here.
```

Comment:
Can we comment out this video section so it doesn't look half done?

Reply:
Good catch. Yes